### PR TITLE
🧹 Fix and cleanup db triggers for broadcasts

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/sql/current_functions.sql [hunk 1]
```diff
@@ -52,60 +52,66 @@ $$ IMMUTABLE LANGUAGE plpgsql;
...
+CREATE OR REPLACE FUNCTION temba_broadcast_determine_system_label(_bcast msgs_broadcast) RETURNS CHAR(1) STABLE AS $$
 BEGIN
-  IF _broadcast.schedule_id IS NOT NULL AND _broadcast.is_active = TRUE THEN
+  IF _bcast.schedule_id IS NOT NULL AND _bcast.is_active = TRUE THEN
     RETURN 'E';
   END IF;
 
-  IF _broadcast.status = 'Q' AND _broadcast.is_active = TRUE THEN
-    RETURN 'O';
-  END IF;
```
Comment (rowanseymour) [inline temba/sql/current_functions.sql:63]: This is wrong and put the Outbox system label count out of sync with the queryset. We do show broadcasts in the outpbox view, but they're fetched and displayed separately at the top of the list view.

File: temba/sql/current_functions.sql [hunk 2]
```diff
@@ -52,60 +52,66 @@ $$ IMMUTABLE LANGUAGE plpgsql;
...
-  IF TG_OP = 'INSERT' THEN
-    _new_label_type := temba_broadcast_determine_system_label(NEW);
-    IF _new_label_type IS NOT NULL THEN
-      PERFORM temba_insert_system_label(NEW.org_id, _new_label_type, 1);
-    END IF;
+    -- add negative system label counts for all rows that belonged to a system label
+    INSERT INTO msgs_systemlabelcount("org_id", "label_type", "count", "is_squashed")
+    SELECT org_id, temba_broadcast_determine_system_label(oldtab), -count(*), FALSE FROM oldtab
+    WHERE temba_broadcast_determine_system_label(oldtab) IS NOT NULL
+    GROUP BY org_id, temba_broadcast_determine_system_label(oldtab);
```
Comment (rowanseymour) [inline temba/sql/current_functions.sql:74]: obviously we don't insert many scheduled broadcasts so there's no performance win here but it's nice having broadcasts follow the same pattern as messages and calls.

File: temba/sql/current_functions.sql (hunk 1 above)
Comment (rowanseymour) [reply temba/sql/current_functions.sql:63]: We use the count on the queryset of msgs.. so conceivably if you had  queued broadcasts but no messages with label=O, that view would blow up.

File: temba/msgs/views.py [hunk 3]
```diff
@@ -657,7 +657,7 @@ def derive_menu(self):
                         menu_id="outbox",
                         name="Outbox",
                         href=reverse("msgs.msg_outbox"),
-                        count=counts[SystemLabel.TYPE_OUTBOX],
+                        count=counts[SystemLabel.TYPE_OUTBOX] + Broadcast.get_queued(org).count(),
```
Comment (rowanseymour) [inline temba/msgs/views.py:660]: Because this view displays both

**Review by norkans7 — APPROVED** [review]

**Pull request merged**